client/my-sites/site-settings/form-general.jsx

```diff
@@ -423,193 +421,42 @@ class SiteSettingsFormGeneral extends Component {
-			<FormFieldset>
-				<FormLabel>
-					{ translate( 'Time Format' ) }
-				</FormLabel>
-				{ defaultFormats.map( ( format, key ) =>
-					<FormLabel key={ key }>
-						<FormRadio
-							checked={ format === time_format }
-							disabled={ isRequestingSettings }
-							name="time_format"
-							onChange={ handleRadio }
-							value={ format }
-						/>
-						<span>{ today.format( phpToMomentDatetimeFormat( format ) ) }</span>
-					</FormLabel>
-				) }
-				<FormLabel className={ customFieldClasses }>
-					<FormRadio
-						checked={ isCustomFormat }
-						disabled={ isRequestingSettings }
-						name="time_format"
-						onChange={ handleRadio }
-						value={ time_format }
-					/>
-					<span>
-						{ translate( 'Custom' ) }
-						<FormInput
-							disabled={ isRequestingSettings }
-							name="time_format_custom"
-							onChange={ onChangeField( 'time_format' ) }
-							type="text"
-							value={ time_format || '' }
-						/>
-						<span className="site-settings__date-time-format-custom-preview">
-							{ isCustomFormat && time_format
-								? today.format( phpToMomentDatetimeFormat( time_format ) )
-								: ''
-							}
-						</span>
-					</span>
-					<FormSettingExplanation>
-						<ExternalLink href="https://codex.wordpress.org/Formatting_Date_and_Time" icon>
-							{ translate( 'Documentation on date and time formatting.' ) }
-						</ExternalLink>
-					</FormSettingExplanation>
-				</FormLabel>
-			</FormFieldset>
-		);
-	}
-
-	startOfWeekOption() {
-		if ( ! config.isEnabled( 'manage/site-settings/date-time-format' ) ) {
-			return null;
-		}
-
-		const {
-			fields: { start_of_week },
-			handleSelect,
-			isRequestingSettings,
+			site,
 			translate,
 		} = this.props;
 
-		const daysOfWeek = [
-			translate( 'Sunday' ),
-			translate( 'Monday' ),
-			translate( 'Tuesday' ),
-			translate( 'Wednesday' ),
-			translate( 'Thursday' ),
-			translate( 'Friday' ),
-			translate( 'Saturday' ),
-		];
+		const now = getNow( timezoneString );
 
 		return (
-			<FormFieldset>
-				<FormLabel>
-					{ translate( 'Week Starts On' ) }
-				</FormLabel>
-				<FormSelect
-					disabled={ isRequestingSettings }
-					name="start_of_week"
-					onChange={ handleSelect }
-					value={ start_of_week || 0 }
-				>
-					{ daysOfWeek.map( ( day, index ) =>
-						<option key={ index } value={ index } >
-							{ day }
-						</option>
-					) }
-				</FormSelect>
-			</FormFieldset>
+			<Card
+				className="site-settings__date-time-format"
+				href={ `/settings/date-time-format/${ site.slug }` }
+			>
+				<h2 className="site-settings__date-time-format-title">
+					{ translate( 'Date and Time Format' ) }
+				</h2>
+				<div className="site-settings__date-time-format-info">
+					{ dateFormat ? now.format( phpToMomentDatetimeFormat( dateFormat ) ) : '' } &bull;&nbsp;
+					{ timeFormat ? now.format( phpToMomentDatetimeFormat( timeFormat ) ) : '' } &bull;&nbsp;
+					{ translate( 'Week starts on' ) } { startOfWeek
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Week Starts On' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).